server.js

```diff
@@ -1,11 +1,135 @@
 const express = require('express')
 const app = express()
+const cors = require('cors')
+const bodyParser = require('body-parser')
+const { Dvr } = require('@mui/icons-material')
 const port = 5000
 
+const sqlite3 = require('sqlite3').verbose();
+
+const db = new sqlite3.Database('./shopListDB.db', sqlite3.OPEN_READWRITE, (err) => {
+  if(err) 
+    return console.error(err.message);
+
+  console.log('connection with database successful');
+});
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use(cors({
```

Comment:
Why do you need this?

Reply:
no need to json.stringify every time
bodyParser.json() does it automatically

Reply:
The comment was on 1 line - line 20
I meant why do you need the `cors`